DistanceSensor/DistanceSensor/main.asm

```diff
@@ -0,0 +1,50 @@
+;
+; DistanceSensor.asm
+;
+; Created: 14/11/2018 10.09.43
+; Author : adand
+;
+
+; Main project, Distance Sensor
+
+INIT:
+	LDI R16, 0x01		; Set TRIG pin to high and output signal, pin 1 in port A contains the high signal.
```

Comment:
I prefer loading a register right before using it, then it's much more clear why you are doing it.